experiments/list_unsupported.sh

```diff
@@ -34,8 +34,30 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+export ADA_INCLUDE_PATH=/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude/
```

Comment:
This would prevent the user of the script from setting their own path if they have tools installed in a different location. Probably better to do something like:
```
export ADA_INCLUDE_PATH=${ADA_INCLUDE_PATH:-/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude/}
```
(I've not checked that syntax, so you may need to check quoting of the path...)

Reply:
But what if neither the user-defined path nor our path exists? I rewrote it a bit differently, but can do your way as well. Your choice.

Reply:
They idiomatic way to do this in most sh scripting is:
```
export FOO={FOO:-"default value for FOO if not already set"}
if [ FOO is not valid ] ; then
   error stuff
fi
```
So I'd suggest doing it that way.

Reply:
Done.